### PR TITLE
fixed broken links, added community forum link, university link, spelling and  other mistakes

### PR DESCRIPTION
**Pull request by guy9**

### CODE REVIEW

**Comment by guy9** [comment]
@annastuchlik please review

File: docs/dev/docker-hub.md [hunk 1]
```diff
@@ -9,25 +9,25 @@ ScyllaDB is released under the GNU Affero General Public License version 3 and t
 
 ![logo](http://www.scylladb.com/wp-content/uploads/mascot_medium.png)
 
-Scylla Docker support x86_64 for all version, and aarch64 starting from Scylla 4.6. Get it now with Scylla [nightly build](https://hub.docker.com/r/scylladb/scylla-nightly)
+ScyllaDB Docker support x86_64 for all version, and aarch64 starting from ScyllaDB 4.6. Get it now with ScyllaDB [nightly build](https://hub.docker.com/r/scylladb/scylla-nightly)
```
Comment (annastuchlik) [inline docs/dev/docker-hub.md:12]:
```suggestion
ScyllaDB Docker supports x86_64 for all versions, and aarch64 starting from ScyllaDB 4.6. Get it now with ScyllaDB [nightly build](https://hub.docker.com/r/scylladb/scylla-nightly)
```

File: docs/dev/docker-hub.md [hunk 2]
```diff
@@ -9,25 +9,25 @@ ScyllaDB is released under the GNU Affero General Public License version 3 and t
...
 
 ## Quick start
 
-To startup a Scylla single-node cluster in developer mode execute:
+To startup a ScyllaDB single-node cluster in developer mode execute:
```
Comment (annastuchlik) [inline docs/dev/docker-hub.md:16]:
```suggestion
To startup a ScyllaDB single-node cluster in developer mode, execute:
```

File: docs/dev/docker-hub.md [hunk 3]
```diff
@@ -9,25 +9,25 @@ ScyllaDB is released under the GNU Affero General Public License version 3 and t
...
 (see `--developer-mode 1`) limited by a single CPU core (see `--smp`).
 Production grade configuration requires tuning a few kernel parameters
-such that limiting number of available cores (with `--smp 1`) is
+such that limiting the number of available cores (with `--smp 1`) is
 the simplest way to go.
 
-Multiple cores requires setting a proper value to the `/proc/sys/fs/aio-max-nr`.
-On many non production systems it will be equal to 65K. The formula
-to calculate proper value is:
+Using multiple cores, requires setting a proper value to the `/proc/sys/fs/aio-max-nr`.
```
Comment (annastuchlik) [inline docs/dev/docker-hub.md:28]:
```suggestion
Using multiple cores requires setting a proper value to the `/proc/sys/fs/aio-max-nr`.
```

File: docs/dev/docker-hub.md [hunk 4]
```diff
@@ -123,54 +123,54 @@ INFO  2016-08-04 06:57:40,840 [shard 0] storage_service - Thrift server listenin
...
-The Scylla docker image defaults to running on overprovisioned mode and won't apply any CPU pinning optimizations, which it normally does in non-containerized environments.
+The ScyllaDB docker image defaults to running on overprovisioned mode and won't apply any CPU pinning optimizations, which it normally does in non-containerized environments.
 For better performance, it is recommended to configure resource limits for your Docker container using the `--smp`, `--memory`, and `--cpuset` command line options, as well as 
 disabling the overprovisioned flag as documented in the section "Command-line options".
 
-### Restart Scylla
+### Restart ScyllaDB
 
-The Docker image uses supervisord to manage Scylla processes. You can restart Scylla in a Docker container using
+The Docker image uses supervisord to manage ScyllaDB processes. You can restart ScyllaDB in a Docker container using
```
Comment (annastuchlik) [inline docs/dev/docker-hub.md:148]:
```suggestion
The Docker image uses a supervisor to manage ScyllaDB processes. You can restart ScyllaDB in a Docker container using:
```

File: docs/dev/docker-hub.md [hunk 5]
```diff
@@ -429,20 +429,23 @@ Example value:
...
+
+For questions and comments, use ScyllaDB's [Community Forum](https://forum.scylladb.com/).
 
 ### Contributing
 
-Want to scratch your own itch and contribute a patch.
+Want to scratch your own itch and contribute a patch?
 We are eager to review and merge your code.
-Please consult the [Contributing on Scylla page](http://www.scylladb.com/kb/contributing/)
+Please consult the [Contributing to ScyllaDB page](https://github.com/scylladb/scylladb/blob/master/CONTRIBUTING.md)
```
Comment (annastuchlik) [inline docs/dev/docker-hub.md:451]:
```suggestion
Please consult the [Contributing to ScyllaDB](https://github.com/scylladb/scylladb/blob/master/CONTRIBUTING.md) page.
```

File: docs/dev/docker-hub.md [hunk 6]
```diff
@@ -429,20 +429,23 @@ Example value:
...
-For bug reports, please use Scylla's [issue tracker](https://github.com/scylladb/scylla/issues) on GitHub.
-Please read the [How to report a Scylla problem](http://docs.scylladb.com/operating-scylla/troubleshooting/report_scylla_problem/) page before you report bugs.
+For bug reports, please use ScyllaDB's [issue tracker](https://github.com/scylladb/scylla/issues) on GitHub.
+Please read the [How to report a ScyllaDB problem](http://docs.scylladb.com/operating-scylla/troubleshooting/report_scylla_problem/) page before you report bugs.
 
-For general help, see Scylla's [documentation](http://www.scylladb.com/doc/).
-For questions and comments, use Scylla's [mailing lists](http://www.scylladb.com/community/).
+For general help, see ScyllaDB's [documentation](https://docs.scylladb.com/stable/).
+
+Training material and online courses can be found at [ScyllaDB University](https://university.scylladb.com/).
```
Comment (annastuchlik) [inline docs/dev/docker-hub.md:443]:
```suggestion
You can find training material and online courses at [ScyllaDB University](https://university.scylladb.com/).
```

File: docs/dev/docker-hub.md (hunk 4 above)
Comment (guy9) [reply docs/dev/docker-hub.md:148]: supervisord is the name of a system, I'll make it bold so it's more clear

**Comment by guy9** [comment]
Thanks @annastuchlik I made the updates.

**Review by annastuchlik — APPROVED** [review]
LGTM

**Comment by annastuchlik** [comment]
@scylladb/scylla-maint Please merge.

**Pull request merged**